GuessingGame.html

```diff
@@ -51,6 +52,105 @@
     } while (total < 3);
     if (total > 2) {
       alert("Congrats! " + name + "! You kicked ass!");
+=======
+<html>
+  <body>
+    <head>
+  <title>Bad Ass Ladies of the Supreme Court</title> </head>
+  <h1>Bad Ass <strong>Ladies</strong> of The United States <strong>Supreme Court</strong></font></h1>
```

Comment:
Your Body should start right below the Head tags, and I would encourage you to encapsulate all of the content tags (such as P, H1, Form, etc) into larger block element tags (such as Header, Main, Article, Section, Footer, etc). 

That will give you more context / capability to manipulate your elements later on, when you get into more dynamic work with JS and CSS.